tools/bazel

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export HERMETIC_CC_TOOLCHAIN_CACHE_PREFIX=/tmp/zig-cache-$USER
```

Comment:
I would like to avoid using tools/bazel if possible as it's not customizable/hard to customize.

Could we set this in `.bazelrc` to some default values, then let folks override it in their `user.bazelrc` with their own value?

Reply:
No, not really. It needs to be a per-user directory to avoid failures on multi-user machines, and the `.bazelrc` cannot do any substitution. I, too, wished to avoid `//bazel/tools` and searched for another solution for hours. If you know of one, let me know! I don't love this, either.

Reply:
Could we patch the toolchain to use the directory we want?

Related but I still prefer a directory in {user}/.caches/bazel/zig-cache so that all build stuff is in one place and easily removed together rather than spread across the system. Can we use this dir?

Reply:
Notably, people can already customize this value in their `user.bazelrc` with `build --action_env HERMETIC_CC_TOOLCHAIN_CACHE_PREFIX=/path/to/directory`. All this does is set a sane default for that environment variable.

Reply:
Fixed.